File: data_diff/src/data_diff/examples.clj
(ns data-diff.examples
  (:require [data-diff.core :as d]))

(defn -main ^{:t :void} [& _args]
      (let [a ^{:t {:vector [:any]}, :cast :any} [1 2]
            b ^{:t {:vector [:any]}, :cast :any} [1 3]
            v (d/diff a b)]
           (println v))
      )

File: data_diff/java/src/datadiff/Examples.java
package datadiff;
import java.util.HashMap;
import java.util.HashSet;
import java.util.List;
import java.util.ArrayList;
import java.util.stream.Collectors;
import kalai.Kalai;
import kalai.Kalai.*;
public class Examples {
public static final void main(String[] _args) {
final Object a = (Object)new io.lacuna.bifurcan.List<Object>().addLast(1L).addLast(2L);
final Object b = (Object)new io.lacuna.bifurcan.List<Object>().addLast(1L).addLast(3L);
final Object v = datadiff.Core.diff(a, b);
System.out.println(v);
}
}
